.changeset/sour-brooms-trade.md

```diff
@@ -0,0 +1,5 @@
+---
+"@comet/cms-admin": major
```

Comment:
you must not make major changes in main branch

Reply:
of course, that was unintentional (and the change is not breaking)